packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -20,8 +20,24 @@ const StyledHeading = styled(Heading)`
   text-shadow: 0 2px 4px rgba(0,0,0,0.5);
 `
 
+const StyledAnchor = styled(Anchor)`
```

Comment:
Just an aside, but maybe this is a candidate for moving into the shared components lib since the standard in the design seems to be using borders instead of text-decoration.